interactive_engine/executor/ir/graph_proxy/src/adapters/gs_store/read_graph.rs

```diff
@@ -759,14 +648,14 @@ fn encode_runtime_e_label<E: StoreEdge>(e: &E) -> LabelId {
 }
 
 #[inline]
-fn encode_runtime_property(prop_id: PropId, prop_val: Property) -> (NameOrId, Object) {
+pub(super) fn encode_runtime_property(prop_id: PropId, prop_val: Property) -> (NameOrId, Object) {
```

Comment:
How about move `encode_runtime_property()` and `encode_runtime_prop_val` into details.rs as well?